app/models/user.rb

```diff
@@ -24,6 +24,8 @@ class User < ApplicationRecord
   after_save :send_welcome_email, if: :needs_welcome_email?
   before_destroy :clear_swap
 
+  include ApplicationHelper
```

Comment:
This actually bloats `User` quite a bit with stuff it doesn't need - could we perhaps put the helper method under a separate `swaps_helper.rb`?

Reply:
Yes, I can do that ... make a lot of sense

Reply:
fix pushed